.github/workflows/changelog_test.yml

```diff
@@ -0,0 +1,38 @@
+name: Changelog check
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  changelog_update:
+    runs-on: ubuntu-latest
+    container:
+      image: alpine:3.14
```

Comment:
I've never used a container in GH actions. What's the function?

Reply:
Still curious about this

Reply:
It looks like the docker image is needed. It's a standard Linux environment that is minimal in size: https://hub.docker.com/_/alpine/